File: abc/abc293/a.py
# https://atcoder.jp/contests/abc293/submissions/39644218

s = list(input())
for i in range(0, len(s) // 2):
  t = s[2 * i - 2]
  s[2 * i - 2] = s[2 * i - 1]
  s[2 * i - 1] = t

for c in s:
  print(c, end = '')

File: abc/abc293/a.cs
// https://atcoder.jp/contests/abc293/submissions/46673427

class A {
  static void Main() {
    string s = Console.ReadLine();
    for(int i = 0; i < s.Length; i += 2)
    {
      Console.Write($"{s[i + 1]}{s[i]}");
    }
  }
}

